openassessment/templates/openassessmentblock/peer/oa_peer_turbo_mode.html

```diff
@@ -52,16 +52,24 @@ <h3 class="peer-assessment__display__title">{% trans "Assessment #" %}
 
                             {% include "openassessmentblock/oa_submission_answer.html" with answer=peer_submission.answer answer_text_label="Your peer's response to the question above:" %}
 
-                            {% if allow_file_upload and peer_file_url %}
+                            {% if file_upload_type and peer_file_url %}
                                 <header class="peer-assessment__display__header">
                                     <h3 class="peer-assessment__display__title">
                                         {% trans "Associated Image" %}
                                     </h3>
                                 </header>
 
-                            <div class="peer-assessment__display__image">
-                                <img class="submission--image" alt="{% trans "The image associated with your peer's submission." %}" src="{{ peer_file_url }}"/>
-                            </div>
+                                <div class="peer-assessment__display__file">
+                                    {% if file_upload_type == "image" %}
+                                    <img class="submission--image"
+                                        alt='{% trans "The image associated with your peer's submission." %}'
+                                        src="{{ peer_file_url }}" />
+                                    {% elif file_upload_type == "file" or file_upload_type == "any" %}
+                                    <a href="{{ file_url }}" class="submission--file">
+                                        {% trans "The file associated with your peer's submission." %}
+                                    </a>
+                                    {% endif %}
+                                </div>
```

Comment:
BTW: this code seems to be here three times.  It may have grown to the point of deserving an include, instead of copying.